### DISCUSSION TITLE
"Habs" --> Unterkünfte

### DISCUSSION THREAD

**Discussion by Qiavo-72** [rjcncpt/StarCitizen-Deutsch-INI#518] [Übersetzungsvorschläge]
Hallo an das fleißige Übersetzerteam,

in allen Beschreibungen (aktueller Standort) des Einlog-Screens steht das Wort "_Hab(s)_". Eigentlich ist Habs die Abkürzung für _Habits_, also zu deutsch _Gewohnheiten_. In SC ist das aber als Abkürzung für die "EZ Habs", also Habitations. Das deutsche Wort, was meiner Meinung nach da am besten passen könnte, ist _UNTERKÜNFTE_.

Und als zweiten Vorschlag: für _TRESPASSING_ wurde von Euch die (völlig korrekte!) Übersetzung _UNERLAUBTES BETRETEN_ (oder war's _UNERLAUBTER ZUTRITT_?) gewählt. Nur taucht diese Meldung ja auch auf, wenn man mit'm Raumschiff durch Teile des Vers düst. Und mit so einem Schiffchen betritt man ja nichts. Vielleicht klingt ja _UNERLAUBTES EINDRINGEN_ etwas passender, auch wenn es nicht die strikte Übersetzung des Wortes ist.

**Comment by rjcncpt** [comment]
Hey. Danke für dein Kommentar und dein nettes Feedback. Wir schauen uns das mal an damit deine Vorschläge in 3.23 dann online kommen 😊

**Comment by rjcncpt** [comment]
Ich habe ein paar Korrekturen vorgenommen die mit 3.23 dann rein kommen 😊

**Reply by Qiavo-72** [reply]
Hi und guten Morgen,

GitHub ist noch neu für mich, muss hier erstmal durchsteigen. 😁
Na, da bin ich ja echt gespannt, ob meine Vorschläge es in die Analen der Localization geschafft haben. 😉

 👋🏼
